gef-ui/server/controllers/amendments/facility-value/get-facility-value.ts

```diff
@@ -0,0 +1,46 @@
+import { CURRENCY, CustomExpressRequest } from '@ukef/dtfs2-common';
+import { Response } from 'express';
+import * as api from '../../../services/api';
+import { FacilityValueViewModel } from '../../../types/view-models/amendments/facility-value-view-model';
+import { asLoggedInUserSession } from '../../../utils/express-session';
+import { getCurrencySymbol } from './getCurrencySymbol';
+import { canUserAmendFacility } from '../../../utils/facility-amendments.helper';
+
+export type GetFacilityValueRequest = CustomExpressRequest<{
+  params: { dealId: string; facilityId: string; amendmentId: string };
+}>;
+
+export const getFacilityValue = async (req: GetFacilityValueRequest, res: Response) => {
+  try {
+    const { dealId, facilityId, amendmentId } = req.params;
+    const { userToken, user } = asLoggedInUserSession(req.session);
+
+    const deal = await api.getApplication({ dealId, userToken });
```

Comment:
Can we add a unit test for this please

Reply:
I think it already exists - gef-ui\server\controllers\amendments\facility-value\get-facility-value.test.ts is there anything else you think should be covered?

Reply:
I can't see a test for it - can we add one with `toHaveBeenCalledTimes(1)`, `CalledWith()` etc?